ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -507,6 +510,7 @@ tests:
       E2E_RUN_TAGS: '@aws-ipi and @disconnected and @rosa'
       OPENSHIFT_VERSION: "4.12"
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
+      TEST_FILTERS_SUPPLEMENTARY: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
```

Comment:
`TEST_FILTERS` use `~ConnectedOnly&;`, while `TEST_FILTERS_SUPPLEMENTARY` use `~DisconnectedOnly&;`. Is it expected ?